src/test/suite/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from 'path'
+import Mocha = require('mocha')
+import glob = require('glob')
+
+export function run() : Promise<void> {
+    // Create the mocha test
+    const mocha = new Mocha({
```

Comment:
Why do we need the manual capturing of test files and running them? Mocha should be able to do that by default.

Reply:
This was all generated as part of building the skeleton. I'd rather just leave it as-is.